biz_portal/apps/portal/templates/portal/search_snippet.html

```diff
@@ -9,7 +9,14 @@
   <input type="hidden" name="region" value="{{ request.GET.region }}" />
   {% endif %}
```

Comment:
is this file still used? If not it should be deleted.

Reply:
Nope. removed